examples/cc26xx/cc26xx-web-demo/project-conf.h

```diff
@@ -53,6 +53,9 @@
 #define NBR_TABLE_CONF_MAX_NEIGHBORS          5
 #define UIP_CONF_MAX_ROUTES                   5
 #define UIP_CONF_TCP_MSS                    128
+/* Additionally disable phase optimization to save RAM */
+#undef CONTIKIMAC_CONF_WITH_PHASE_OPTIMIZATION
```

Comment:
Seemy previous commit - I think it ran out of RAM when trying to link the web demo with phase lock enabled.